sycl/CMakeLists.txt

```diff
@@ -84,11 +84,16 @@ if( NOT OpenCL_LIBRARIES )
   if(MSVC)
     set(OpenCL_LIBRARIES
         "${LLVM_LIBRARY_OUTPUT_INTDIR}/${CMAKE_STATIC_LIBRARY_PREFIX}OpenCL${CMAKE_STATIC_LIBRARY_SUFFIX}")
-    set(OPENCL_ICD_LOADER_WDK "-DOPENCL_ICD_LOADER_REQUIRE_WDK=OFF")
+    set(AUX_CMAKE_FLAGS "-DOPENCL_ICD_LOADER_REQUIRE_WDK=OFF")
```

Comment:
```suggestion
    set(AUX_CMAKE_FLAGS "${AUX_CMAKE_FLAGS} -DOPENCL_ICD_LOADER_REQUIRE_WDK=OFF")
```

Reply:
Looks good to me, please just make a fix for Alexey's comment (3 days ago).